stdlib/public/SwiftRemoteMirror/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+append("-undefined dynamic_lookup" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
```

Comment:
Whats with this part of the change here?

Reply:
Ugh, somehow I missed this part of the change.  This seems wrong; I think that it is trying to work around the missing `LLVMSupport` linkage.  Thank you @gottesmm for catching this oversight.